cloud_governance/tag_cluster/remove_cluster_tags.py

```diff
@@ -156,12 +163,17 @@ def cluster_instance(self):
         instance_list = self.__get_instances_data()
         cluster, non_cluster = self.ec2_operations.scan_cluster_or_non_cluster_instance(instance_list)
         queue = Queue()
-        cluster_process = Process(target=self.cluster_update_tags, args=(cluster, queue, ))
-        cluster_process.start()
-        non_cluster_process = Process(target=self.non_cluster_update.non_cluster_update_ec2, args=(non_cluster, ))
-        non_cluster_process.start()
-        cluster_process.join()
-        non_cluster_process.join()
+        if not self.cluster_only:
```

Comment:
Why are you using 'cluster_only' term and not 'cluster' and 'non_cluster' ?

Reply:
I used variable cluster so I used cluster only.